engine/src/main/java/org/terasology/logic/behavior/nui/BehaviorEditor.java

```diff
@@ -57,12 +58,12 @@
 
     private final InteractionListener moveOver = new BaseInteractionListener() {
         @Override
-        public void onMouseOver(Vector2i pos, boolean topMostElement) {
+        public void onMouseOver(Vector2i pos, boolean topMostElement, KeyboardDevice keyboard) {
```

Comment:
I would suggest creating a MouseEvent class or similar to hold these bits of information, so that it can be extended as necessary without having to change the method signature again.